src/containers/FileExplorer/FileExplorer.tsx

```diff
@@ -379,13 +384,22 @@ class MapModelToAssetForm extends React.Component<Props, State> {
               </div>
             );
           })}
+        <Pagination
```

Comment:
Can we reuse this object as it is the same below?

Reply:
When you press resolve, can you also just write a quick reply like `Fixed` or so? Now I don't know what happened without looking through the code again.